packages/ezviewer.vm/ezviewer.vm.nuspec

```
@@ -2,13 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>ezviewer.vm</id>
-    <version>2.0.0.20250219</version>
+    <version>2.0.0.20250430</version>
     <authors>Eric Zimmerman</authors>
-    <description>Standalone, zero dependency viewer for .doc, .docx, .xls, .xlsx, .txt, .log, .rtf, .otd, .htm, .html, .mht, .csv, and .pdf. Any non-supported files are shown in a hex editor (with data interpreter!)</description>
+    <description>Ezviewer is a standalone, zero dependency viewer for .doc, .docx, .xls, .xlsx, .txt, .log, .rtf, .otd, .htm, .html, .mht, .csv, and .pdf files</description>
```

This description is too long and includes specific details and other information that is likely to change in a future version not following our documentation. Please address this kind of issues when improving the descriptions.

```suggestion
<description>Ezviewer is a standalone, zero dependency document viewer and hex editor.</description>
```